### PR TITLE
andOTP rewordings [Improved English Strings]

### PR DESCRIPTION
**Pull request by enwokoma**
Cleaned up andOTP wordings as proposed by @flocke in #149 and promised in https://github.com/andOTP/andOTP/issues/149#issuecomment-370519141

### CODE REVIEW

**Comment by enwokoma** [comment]
Sorry if the PR is coming late, I had a lot on my schedule these days! 
I've still got a tight schedule all through this month and probably next month too! 

But I would try to make more reviews on the strings in andOTP when am less busy, for now, this are the major areas I think that need some rewording!

File: app/src/main/res/values/strings_about.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <resources>
     <string name="about_activity_title">About</string>
 
-    <string name="about_description">Open source two-factor authentication for Android</string>
```
Comment (flocke) [inline app/src/main/res/values/strings_about.xml]: I would like to keep this as it is now, it is kind of the tagline for andOTP

File: app/src/main/res/values/strings_about.xml [hunk 2]
```diff
@@ -37,15 +37,15 @@
...
-        features? Most of them are non-standard OTP algorithms which were implemented on special
+        features of andOTP? Most of them are non-standard OTP algorithms which were implemented on special
         request. They are <b>not officially supported</b> and can come with <b>certain
         limitations</b>, please read (and understand) the page \"Special features\" in the Github
-        wiki to learn more before using them.</string>
-    <string name="about_toast_special_features">Special features enabled</string>
-    <string name="about_toast_special_features_enabled">Special features already enabled</string>
-    <string name="about_toast_copied_to_clipboard">Donation link copied to clipboard</string>
+        wiki so as to learn more about these features before using them.</string>
+    <string name="about_toast_special_features">Special features has been enabled</string>
```
Comment (flocke) [inline app/src/main/res/values/strings_about.xml]: I think it should be `have been` (plural)

File: app/src/main/res/values/strings_intro.xml [hunk 3]
```diff
@@ -3,31 +3,32 @@
...
         stores them in encrypted data files. Here you can choose which method of encryption will be
         used.</string>
     <string name="intro_slide2_desc_keystore">The KeyStore is a system component of Android for
-        securely storing cryptographic keys. The advantage of this method is that the keys are
-        stored separated from the data files and can be backed by hardware cryptography (if your
-        device supports it). However, as the keys are not stored with the apps data, <b>this method
-        prevents external backup solutions (like Titanium) from working</b>. If you choose this
-        method you will have to rely on the internal backup functions provided by <b>andOTP</b>.
+        securely storing cryptographic keys. The advantage of using this method is that the keys are
+        stored separated from the data files of andOTP and can be backed up by hardware cryptography (if your
```
Comment (flocke) [inline app/src/main/res/values/strings_intro.xml]: I don't think `backed up` is the correct wording in this context, `backed by hardware cryptography` is the correct wording (in my opinion). Maybe @RichyHBM can chime in as a third opinion.

File: app/src/main/res/values/strings_intro.xml [hunk 4]
```diff
@@ -3,31 +3,32 @@
...
+        device supports it). However, as the keys are not stored with the andOTP\'s data files, <b>this method
```
Comment (flocke) [inline app/src/main/res/values/strings_intro.xml]: We should probably remove the `the` if we use `andOTP's`.

File: app/src/main/res/values/strings_main.xml [hunk 5]
```diff
@@ -66,14 +66,14 @@
...
 
-    <string name="dialog_msg_auth">Please enter your device credentials to start andOTP.</string>
+    <string name="dialog_msg_auth">Please enter your device credentials in order to start andOTP.</string>
     <string name="dialog_msg_confirm_delete">Are you sure you want do remove the account \"%1$s\"?</string>
 
-    <string name="dialog_msg_last_used">In order for andOTP to recognize which token was used last
-        you have to have \"tap to reveal\" enabled or use the copy button.\n\nThis message will not
-        be shown again.</string>
+    <string name="dialog_msg_last_used">In order for andOTP to recognize which token was last used,
+        you will have to enable <b>\"tap to reveal\"</b> feature in the <b>Settings</b> page,
```
Comment (flocke) [inline app/src/main/res/values/strings_main.xml]: We should add a `the` before `<b>\"tap to reveal\"</b> feature`

File: app/src/main/res/values/strings_backup.xml [hunk 6]
```diff
@@ -22,26 +22,26 @@
...
-    <string name="backup_desc_openpgp_provider">You need to install an OpenPGP provider and enable
-        it in the <b>Settings</b> to use this feature.
+    <string name="backup_desc_openpgp_provider">Please, you need to install an OpenPGP provider and enable
+        it in the <b>Settings</b> page in order to use this feature.
     </string>
-    <string name="backup_desc_openpgp_keyid">You need to select an OpenPGP key in the <b>Settings</b>
-        before you can create encrypted backups.
+    <string name="backup_desc_openpgp_keyid">Please, you need to select an OpenPGP key in the <b>Settings</b>
+        page before you can create encrypted backups.
     </string>
```
Comment (flocke) [inline app/src/main/res/values/strings_backup.xml]: I'm not sure about the `Please, you` in the three strings above. In my opinion we should either keep it as only `You need to ...` or drop the `you need to` completely and only use the `Please` (e.g. `Please install an OpenPGP provider ...`). Maybe @RichyHBM can chime in here as well for a third opinion.

File: app/src/main/res/values/strings_intro.xml (hunk 3 above)
Comment (RichyHBM) [reply app/src/main/res/values/strings_intro.xml]: Yea, the previous version makes more grammatical sense to me

File: app/src/main/res/values/strings_backup.xml (hunk 6 above)
Comment (RichyHBM) [reply app/src/main/res/values/strings_backup.xml]: Agree with @flocke

File: app/src/main/res/values/strings_about.xml (hunk 2 above)
Comment (Maikuolan) [reply app/src/main/res/values/strings_about.xml]: > I think it should be `have been` (plural)

A native English speaker here. You are correct.

**Comment by enwokoma** [comment]
Wow. Unbelievable! I totally forgot about this PR. 
My apologies @flocke and @RichyHBM and all other contributors. I had a lot of work to handle over the past months. 
But anyways, now that I've remembered this PR, I think am going to fix it up anytime soon! 

My apologies once again for the delay!

File: app/src/main/res/values/strings_backup.xml (hunk 6 above)
Comment (enwokoma) [reply app/src/main/res/values/strings_backup.xml]: Ok. Point noted. Would effect the chances ASAP

File: app/src/main/res/values/strings_main.xml (hunk 5 above)
Comment (enwokoma) [reply app/src/main/res/values/strings_main.xml]: Ok. Point noted. Would effect the chances ASAP

File: app/src/main/res/values/strings_intro.xml (hunk 4 above)
Comment (enwokoma) [reply app/src/main/res/values/strings_intro.xml]: Ok. Point noted. Would effect the chances ASAP

File: app/src/main/res/values/strings_intro.xml (hunk 3 above)
Comment (enwokoma) [reply app/src/main/res/values/strings_intro.xml]: Ok. Point noted. Would effect the chances ASAP

File: app/src/main/res/values/strings_about.xml (hunk 2 above)
Comment (enwokoma) [reply app/src/main/res/values/strings_about.xml]: Ok @Maikuolan . Point noted. Would effect the chances ASAP

This particular error was typographical. 
My apologies!

File: app/src/main/res/values/strings_about.xml (hunk 1 above)
Comment (enwokoma) [reply app/src/main/res/values/strings_about.xml]: Oh. Ok! 
Didn't know about that! 

Point noted. Would effect the chances ASAP

**Comment by flocke** [comment]
Closing since this has been open for way to long now and a lot has changed since then.

**Pull request closed without merging**